### DISCUSSION TITLE
GSA Shop - Dino cryopod

### DISCUSSION THREAD

**Discussion by Mrtritolo** [gameserverapp/Platform#679] [Suggestions]
Good morning, I don't know if it's possible or not to do this but it would be nice to have a system that allows you to buy dino in cryopod as it is on the arkshop plugin

**Comment by mvaessen** [comment]
Please submit suggestions for plugins with plugin devs :)

**Reply by Mrtritolo** [reply]
Mine was not a request for a plugin but an example of a feature that would be nice to have in the GSA mod, I made a comparison with the plugin to give an idea of what I meant

**Reply by mvaessen** [reply]
There are mods that can do this already. I recommend checking out the Dino Depot's Dinoballs mod from Delilah. See https://discord.com/channels/315049169071112193/1169400175505182851
